cli/src/command/host.rs

```diff
@@ -140,6 +140,8 @@ fn setup_crate(host_path: PathBuf, rev: Option<String>, tag: Option<String>) ->
             "# Generated by cargo-nexus, do not remove!\n",
             "#\n",
             "# This profile is used for generating proofs, as Nexus VM support for compiler optimizations is still under development.\n",
```

Comment:
This line shouldn't be here anymore, might as well take the opportunity to remove it.

Reply:
removed in e67de56